index.js

```diff
@@ -64,27 +76,8 @@ class PhotoViewManager {
     return scale;
   }
 
-  imageTouchHandler(e) {
```

Comment:
@shidhincr Yes we did fix that on lory disabling touchstart and touchmove events. But I think emitting an event is a better approach with regards to delegation.